<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="workareaLayout">
      <item>
       <widget class="QListWidget" name="filesWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="filesContent">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="prevOccurrenceButton">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextOccurrenceButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QToolButton" name="changeDirButton">
        <property name="text">
         <string>Change dir</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Current Directory:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1" rowspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="stringInput">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Input string you want to find</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>filesWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>getFileContent(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeDirButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextOccurrenceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>nextOccurrence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevOccurrenceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>previousOccurrence()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopActions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stringInput</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>findSubstring()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findSubstring()</slot>
  <slot>getFileContent(QListWidgetItem*)</slot>
  <slot>changeDirectory()</slot>
  <slot>previousOccurrence()</slot>
  <slot>nextOccurrence()</slot>
  <slot>stopActions()</slot>
 </slots>
</ui>
